readthedocs/audit/models.py

```diff
@@ -117,6 +117,27 @@ class AuditLog(TimeStampedModel):
         db_index=True,
     )
 
+    organization = models.ForeignKey(
+        'organizations.Organization',
+        verbose_name=_('Organization'),
+        null=True,
+        db_index=True,
+        on_delete=models.SET_NULL,
+    )
+    log_organization_id = models.IntegerField(
+        _('Organization ID'),
+        blank=True,
+        null=True,
+        db_index=True,
+    )
```

Comment:
Why is the organization id relevant? Shouldn't be enough with the organization slug?

Reply:
We want to keep track of the organization after it has been deleted. The id is the source of truth, as another organization can be created with the same slug. We save the slug to make it more friendly to recognize the original organization or display.

Reply:
This should be a comment in the code, not in a PR ;)